qiskit/optimization/infinity.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Infinity constant."""
+
+
+infinity = 1.0E+20  # pylint: disable=invalid-name
```

Comment:
This should be capitalized to conform to PEP8 https://www.python.org/dev/peps/pep-0008/#constants and make it clear its a constant. Would this file be better named constants? in case more constants are wanted in the future